sonarr/qualitydefinition.go

```diff
@@ -0,0 +1,74 @@
+package sonarr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"golift.io/starr"
+)
+
+// QualityDefinition is the /api/v3/qualitydefinition endpoint.
+type QualityDefinition struct {
+	ID        int64            `json:"id,omitempty"`
+	Weight    int64            `json:"weight"`
+	MinSize   float64          `json:"minSize"`
+	MaxSize   float64          `json:"maxSize"`
+	Title     string           `json:"title"`
+	Qualities []*starr.Quality `json:"items"`
```

Comment:
@Fuochi,

I'm about to write code that uses quality definitions, so I was looking this over more closely. I do not see `items` on the actual data object in the docs, or in the API responses. I do see `quality` (which is not a list). Any idea why I'm seeing a discrepancy here?

https://sonarr.tv/docs/api/#/QualityDefinition/get_api_v3_qualitydefinition__id_

Reply:
yes, my bad. This was before we started writing tests, otherwise we would've noticed this in PR. It was probably a copy-paste error from QualityProfile

Reply:
No worries. I'll need this in radarr too, so when I get around to it, I'll make sure this one is corrected. Thanks for confirming!

Reply:
if you can add tests as well, it would be great! when I've time I'll write tests for the various profiles as well!

Reply:
I'm pretty bad about writing tests. I just have too damn many projects to make the time.

Reply:
If you want, we can look at something like [gotests](https://github.com/cweill/gotests), it allows to write template for testing. That will save us a lot of time, since most of our tests are similar. We'll still need to write the body to test and the resulting struct, but it will still be faster.